hass_nabucasa/files.py

```diff
@@ -194,35 +199,30 @@ async def download(
 
         return response.content
 
+    @api_exception_handler(FilesError)
     async def list(
         self,
         storage_type: StorageType,
     ) -> list[StoredFile]:
         """List files."""
-        _LOGGER.debug("Listing %s files", storage_type)
```

Comment:
Should we keep the debug log for the sake of consistency?

Reply:
Nah, this is covered by https://github.com/NabuCasa/hass-nabucasa/pull/908